rust/datafusion/src/physical_plan/mod.rs

```diff
@@ -89,6 +89,8 @@ pub trait ExecutionPlan: Debug + Send + Sync {
 pub enum Partitioning {
     /// Unknown partitioning scheme
     UnknownPartitioning(usize),
+    /// parts hashed by columns and `N` partitions
+    HashPartitioning(Vec<String>, usize),
```

Comment:
It makes sense for us to add a `HashPartitioning` variant here, but this isn't required for implementing the hash join operator since we aren't changing partitioning.

Reply:
You are right, I was working on the repartition and join at the same time at some point, and this slipped through the cracks.